### DISCUSSION TITLE
Discard corrupted carved items

### DISCUSSION THREAD

**Discussion by reacan** [sepinf-inc/IPED#414] [Q&A] [answered]
Greetings!

In the processing GUI there is an entry called "Carved discarderd", so I'm wondering if it is possible to exclude corrupted carved items from the case? Looked all over config files, even did a string search but couldn't figure this out.

**Comment by lfcnassif** [comment, marked as answer]
I have just put this info in the manual: https://github.com/sepinf-inc/IPED/wiki/User-Manual#data-carving

Check if it is clear please.

**Comment by reacan** [comment]
@lfcnassif  thank you very much, everything is clear!

**Comment by lfcnassif** [comment]
You're welcome.
